src/ui/templates/search/search.hbs

```diff
@@ -98,24 +112,44 @@
         </button>
         <button {{#if useForm}}type="submit"{{else}}type="button"{{/if}}
           class="js-yext-submit yxt-SearchBar-button">
+          {{#if showLoadingIndicator}}
+            {{#if customLoadingIconUrl}}
+              <div class="yxt-SearchBar-CustomLoadingImage js-yxt-SearchBar-CustomLoadingImage js-yxt-SearchBar-Icon yxt-SearchBar-Icon--inactive">
+                {{> icons/iconPartial iconUrl=customLoadingIconUrl}}
+              </div>
+            {{else}}
+              <div class="yxt-SearchBar-AnimatedIcon js-yxt-SearchBar-LoadingIndicator yxt-SearchBar-LoadingIndicator js-yxt-SearchBar-Icon 
+                yxt-SearchBar-LoadingIndicator--inactive Icon--lg">
+                <svg class= "yxt-SearchBar-LoadingIndicator-AnimatedIcon" viewBox="0 0 72 72">
+                  <circle class="yxt-SearchBar-LoadingIndicator-Circle" cx="36" cy="36" r="33" stroke="black" stroke-width="3" fill="none"/>
+                </svg>
+              </div>
+            {{/if}}
+          {{/if}}
           {{#if submitIcon}}
-            <div class="yxt-SearchBar-buttonImage">
+            <div class="yxt-SearchBar-buttonImage js-yxt-SearchBar-buttonImage js-yxt-SearchBar-Icon">
               {{> icons/bakedInIcon iconName=submitIcon}}
             </div>
           {{else if _config.customIconUrl}}
-            <div class="yxt-SearchBar-buttonImage">
+            <div class="yxt-SearchBar-buttonImage js-yxt-SearchBar-buttonImage js-yxt-SearchBar-Icon">
               {{> icons/iconPartial iconUrl=_config.customIconUrl}}
             </div>
           {{else}}
-            <div class="yxt-SearchBar-AnimatedIcon yxt-SearchBar-AnimatedIcon--paused js-yxt-AnimatedForward
-              {{#if autoFocus}} yxt-SearchBar-AnimatedIcon--inactive{{/if}}"
-              data-component="IconComponent"
-              data-opts="{{json forwardIconOpts}}">
+            <div class="yxt-SearchBar-AnimatedIcon yxt-SearchBar-AnimatedIcon--paused js-yxt-AnimatedForward js-yxt-SearchBar-Icon
+              {{#unless autoFocus}} yxt-SearchBar-Icon--inactive{{/unless}}">
+              {{> icons/bakedInIcon
```

Comment:
(I just renamed this to builtInIcon after some comments from Tom sorry about affecting your pr :( )

Reply:
No worries! I will update it, thanks for letting me know!